feedthefox/base/templates/jinja2/base.html

```diff
@@ -38,7 +38,19 @@
             <div class="collapse navbar-collapse navbar-left" id="navbar-collapse">
               <ul class="nav navbar-nav">
                 <li><a href="{{ url('devices') }}">Devices</a></li>
-                <li><a href="{{ url('dashboard') }}">Dashboard</a></li>
+                <li class="dropdown">
+                  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button"
+                      aria-haspopup="true" aria-expanded="false">
+                    Resources
+                  </a>
+                  <ul class="dropdown-menu">
+                    <li><a href="{{ url('porting') }}">Porting Guide</a></li>
+                    <li><a href="{{ url('gaia') }}">Gaia Development</a></li>
+                    <li><a href="{{ url('addons') }}">Add-ons</a></li>
+                    <li><a href="{{ url('b2gdroid') }}">B2GDroid</a></li>
+                  </ul>
+                </li>
+                <li class="hidden-sm"><a href="{{ url('dashboard') }}">Dashboard</a></li>
```

Comment:
That's expected behavior. We only hide this on tablet resolutions where tabzilla minimizes available navbar space.